src/Codeception/Module/WebDriver.php

```diff
@@ -979,8 +983,10 @@ protected function proceedSeeInField(array $elements, $value)
                     break;
 
                 case 'textarea':
+                    // we include trimmed and real value of textarea for check
+                    $currentValues[] = $el->getText(); // trimmed value
```

Comment:
it's actually not.
$currentValues will contain both: '  text value   ' and 'text value' strings, so both checks will be correct:

```php
$I->seeInField('textarea', ' field value ');
$I->seeInField('textarea', 'field value');
```